.github/workflows/goreleaser-ci.yml

```diff
@@ -0,0 +1,35 @@
+
+name: GoReleaser CI
+
+on:
+  push:
```

Comment:
Would be pretty cool to have it run -- and actually sign and upload artifacts, even though we don't use them in prod -- on pushes of new (release) tags. That might be future work though? If so, file a ticket for it somewhere.

Reply:
Yeah I've been suggesting that we do all of our builds outside of our local environment and mirror them internally. I would like to move to that in the future. I'll file a ticket for it.

Reply:
Issue filed internally